### ISSUE TITLE
 [Legitimate Site Blocked] 

### ISSUE THREAD

**Issue by SlavaKPSS31** [MetaMask/eth-phishing-detect#7525]
[Legitimate Site Blocked] binances.club
unblock https://binances.club/

**Comment by devenblake** [comment]
Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/6696.
